projects/packages/videopress/src/class-status.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * The class that provides information about VideoPress Status
+ *
+ * @package automattic/jetpack-videopress
+ */
+
+namespace Automattic\Jetpack\VideoPress;
+
+use Jetpack;
+
+/**
+ * The class that provides information about VideoPress Status
+ */
+class Status {
+
+	/**
+	 * Returns whether VideoPress is active either as a Jetpack module or as a stand alone plugin
+	 *
+	 * @return boolean
+	 */
+	public static function is_active() {
+		return self::is_jetpack_active() || self::is_standalone_plugin_active();
+	}
+
+	/**
+	 * Returns whether the Jetpack plugin and its VideoPress module are active
+	 *
+	 * @return boolean
+	 */
+	public static function is_jetpack_active() {
+		return class_exists( 'Jetpack' ) && Jetpack::is_module_active( 'videopress' );
+	}
+
+	/**
+	 * Checks whether the VideoPress stand alone plugin is active
+	 *
+	 * @return boolean
+	 */
+	public static function is_standalone_plugin_active() {
+		$possible_folders = array(
+			'videopress', // monorepo.
+			'jetpack-videopress', // production.
+			'jetpack-videopress-dev', // Jetpack Beta.
+		);
+		foreach ( $possible_folders as $folder ) {
+			if ( is_plugin_active( $folder . '/jetpack-videopress.php' ) ) {
```

Comment:
Getting a Fatal here:

```
[04-Aug-2022 21:36:26 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function Automattic\Jetpack\VideoPress\is_plugin_active() in /usr/local/src/jetpack-monorepo/projects/packages/videopress/src/class-status.php:50
```

```suggestion
			if ( Jetpack::is_plugin_active( $folder . '/jetpack-videopress.php' ) ) {
```

could ⬆️ work?

Reply:
I ran all the tests again now and I dont see this error. In what situation exactly it happened to you?

Reply:
What we could do instead if the problem persists is `class_exists( 'Jetpack_VideoPress_Plugin' ); `

Reply:
well, I could not reproduce it for some reason, but I was seeing the tests failing, so I changed the way we do the check in https://github.com/Automattic/jetpack/pull/25385/commits/76f39e8f3748ece4ffade5cc357ac0665519357d

Reply:
it should work now